### PR TITLE
Synthtech importer hotfix

### PR DESCRIPTION
**Pull request by vevix**
Extension of #643.

Was going to manually merge the referenced PR but turns out there were more issues than I originally thought so could use some other eyes.

### CODE REVIEW

File: test/fixtures/webmock/mal_anime_11757_characters.response [hunk 1]
```diff
@@ -1,287 +1,418 @@
...
-Transfer-Encoding: chunked
+Cache-Control: no-cache
 Content-Type: text/html; charset=utf-8
-Set-Cookie: incap_ses_225_81958=WzBqBAH+gxmBefsZe1wfAyTtqFMAAAAA9hJiZB75ZmAUFl+rkRkZNA==; path=/; Domain=.myanimelist.net
-Set-Cookie: visid_incap_81958=0QxXheQkT9W1ZPT7esuH5RntqFMAAAAAQUIPAAAAAAAyyfPc7M+vU8PEBVyP0YBo; expires=Wed, 22 Jun 2016 12:53:40 GMT; path=/; Domain=.myanimelist.net
-X-Iinfo: 9-67445001-67445002 NNNN CT(2 -1 0) RT(1403579684177 0) q(0 0 0 0) r(1 1)
+Date: Thu, 17 Sep 2015 01:01:05 GMT
+Server: Apache
+Set-Cookie: MALHLOGSESSID=064de82cd3b3e3d38c1028b7a1116cf8; expires=Tue, 15-Sep-2020 01:01:05 GMT; Max-Age=157680000; path=/
+Set-Cookie: MALSESSIONID=1neoo75lr6gue57fvtuicofue0; expires=Thu, 24-Sep-2015 01:01:05 GMT; Max-Age=604800; path=/; HttpOnly
```
Comment (NuckChorris) [inline test/fixtures/webmock/mal_anime_11757_characters.response]: You didn't just log us into your account here, right?

Comment (vevix) [inline test/fixtures/webmock/mal_anime_11757_characters.response]: Nope don't have an active session and this was just a `curl -i` dump

**Comment by vevix** [comment]
Re-opened. Going to take a look at this again, so mods can keep the site running while we do secret stuff.

**Comment by vevix** [comment]
Good to go :+1: /cc @NuckChorris 

```ruby
MALImport.new(:anime, 5114).metadata
=> {:external_id=>5114,
 :title=>
  {:canonical=>"Fullmetal Alchemist: Brotherhood",
   :unknown=>["Hagane no Renkinjutsushi (2009)", "Fullmetal Alchemist (2009)", "FMA"],
   :en_us=>nil,
   :ja_jp=>"鋼の錬金術師"},
 :synopsis=>
  "In this world there exist alchemists, people who study and perform the art of alchemical transmutation—to manipulate objects and transform one object into another. They are bounded by the basic law of alchemy: in order to gain something you have to sacrifice something of the same value.\r\n\r\nThe main character is the famous alchemist Edward Elric—also known as the Fullmetal Alchemist—who almost lost his little brother, Alphonse, in an alchemical accident. Edward managed to attach his brother's soul to a large suit of armor. While he did manage to save his brother's life, he paid the terrible price of his limbs.\r\n\r\nTo get back what they've lost, the brothers embark on a journey to find the Philosopher's Stone that is said to amplify the powers of an alchemist enormously; however, on the way, they start uncovering a conspiracy that could endanger the entire nation, and they realize the misfortunes brought upon by the Philosopher's Stone.\r\n\r\nFullmetal Alchemist: Brotherhood is a re-telling of the story from the manga's point of view.",
 :poster_image=>#<URI::HTTP:0x007f89abfc9ba8 URL:http://cdn.myanimelist.net/images/anime/5/47421l.jpg>,
 :type=>"TV",
 :status=>"Finished Airing",
 :genres=>["Action", "Adventure", "Drama", "Fantasy", "Magic", "Military", "Shounen"],
 :dates=>[Sun, 05 Apr 2009, Sun, 04 Jul 2010],
 :producers=>["Bones", "Aniplex", "Square Enix", "FUNimation Entertainment", "Mainichi Broadcasting", "Studio Moriken"],
 :age_rating=>"R17+",
 :age_rating_guide=>"Violence, Profanity",
 :episode_count=>64,
 :episode_length=>24}
```

**Comment by NuckChorris** [comment]
LGTM! :thumbsup:

Also, was Brotherhood really 64 episodes!?

**Comment by wopian** [comment]
[It was always 64 episodes ;) ](https://en.wikipedia.org/wiki/List_of_Fullmetal_Alchemist:_Brotherhood_episodes#Episodes_list)

**Pull request merged**